Doc/reference/expressions.rst

```diff
@@ -1874,10 +1878,12 @@ Expression lists
    single: , (comma); expression list
 
 .. productionlist:: python-grammar
-   expression_list: `expression` ("," `expression`)* [","]
-   starred_list: `starred_item` ("," `starred_item`)* [","]
-   starred_expression: `expression` | (`starred_item` ",")* [`starred_item`]
-   starred_item: `assignment_expression` | "*" `or_expr`
+   starred_expression: ["*"] `or_expr`
+   flexible_expression: `assignment_expression` | `starred_expression`
+   expression_list: `flexible_expression` ("," `flexible_expression`)* [","]
+   starred_expression_list: `starred_expression` ("," `starred_expression`)* [","]
+   yield_list: `expression` ["," `starred_expression_list`]
+               | `starred_expression` "," [`starred_expression_list`]
```

Comment:
The second line doesn't get rendered. I think this can be simplified as:
```suggestion
   yield_list: `expression` | `starred_expression` "," [`starred_expression_list`]
```

Reply:
Oops, good catch - and nice simplification!